src/org/usfirst/frc4904/robot/subsystems/MilkIO.java

```diff
@@ -0,0 +1,26 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class MilkIO extends Subsystem {
+
+	public static Motor roller;
+	public static final double INTAKE_SPEED = 1.0;
+	public static final double OUTTAKE_SPEED = -1.0;
+	public static final double AT_REST = 0.0;
+	
+	public MilkIO(Motor roller) {
+		this.roller = roller;
+	}
+	
+	@Override
+	protected void initDefaultCommand() {
+		set(MilkIO.AT_REST);
```

Comment:
This should not be done through a method in this class. This should be done through a special MilkIO rest command that's defined elsewhere, or maybe if this is a very stateful subsystem through an enum with a command defined in the constructor.

Reply:
Put differently, rather than calling `set()` from here, you need to make a `Command` class that does that. Then, you can in this method call `setDefaultCommand(new AtRestCommand());`